scripts/generatePackage.js

```diff
@@ -0,0 +1,70 @@
+import fsExtra from 'fs-extra';
+import fs from 'fs';
+import initPromptSync from 'prompt-sync';
+import { execSync } from 'child_process';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import sortPackageJson from 'sort-package-json';
```

Comment:
I cannot see this `sortPackageJson` used. Is it used?

Reply:
hmm, good catch I wonder why eslint didn't complain, I need to verify settings

I need this to provide unified structure of package.json, in future I plan to add script that will crawl over whole repo and sort all `package.json` files, sort dependencies alphabeticaly etc.

Reply:
it is used now